build/parents/integrationtest/pom.xml

```diff
@@ -493,6 +484,20 @@
             </properties>
         </profile>
 
+        <profile>
+            <id>elasticsearch-container-disable-useless-features</id>
```

Comment:
Mmmm, so this one is a bit confusing... 
from what it looks, we'd always pass the env file as long as we start an ES container.

Reply:
Ah right, that was an earlier attempt, I should remove it.